MAINTAINERS.md

```diff
@@ -165,13 +166,23 @@ When necessary, be upfront that the time to review, approve, and implement a RFC
 
 Some examples using our initial and new RFC templates: #92, #94, #95, #991, #1226
 
+### Run end to end tests on merges to Develop branch
+
+In order to run end to end tests you need to install CDK CLI first and bootstrap your account with `cdk bootstrap` command. For additional details follow [documentation](https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html).
+
+You can run tests either from your local machine's shell by exporting `AWS_PROFILE` environment variable and running `make e2e tests`. 
+You may also do this directly from GitHub Console. Run the [run-e2e-tests workflow](https://github.com/awslabs/aws-lambda-powertools-python/actions/workflows/run-e2e-tests.yml) and pick the branch you want to run tests against.
+
+> **Q: What if there's an error?**
+To be agreed
+
 ### Releasing a new version
 
 > TODO: This is an area we want to increase automation while keeping communication at human level.
 
 Firstly, make sure you are using the `develop` branch and it is up to date with the origin.
 
-There are three main steps to release a new version: Changelog generation, version bumping, and drafting release notes.
+There are four main steps to release a new version: Changelog generation, version bumping, and drafting release notes.
```

Comment:
```suggestion
There are three main steps to release a new version: Changelog generation, version bumping, and drafting release notes.
```

Returning it. In the near future, we'll be moving to a single step: `drafting release notes`.